openmc/lattice.py

```diff
@@ -1059,15 +1139,20 @@ def find_element(self, point):
         else:
             z = point[2] - self.center[2]
             iz = floor(z/self.pitch[1] + 0.5*self.num_axial)
-        alpha = y - x/sqrt(3.)
-        ix = floor(x/(sqrt(0.75) * self.pitch[0]))
-        ia = floor(alpha/self.pitch[0])
-
+        if self._hextype:
+            alpha = y - x*sqrt(3.)
+            i1 = floor(-alpha/(sqrt(3.0) * self.pitch[0]))
+            i2 = floor(y/(sqrt(0.75) * self.pitch[0]))
+        else:
+            alpha = y - x/sqrt(3.)
+            i1 = floor(x/(sqrt(0.75) * self.pitch[0]))
+            i2 = floor(alpha/self.pitch[0])
         # Check four lattice elements to see which one is closest based on local
         # coordinates
+        numbersaxy=[(i1, i2, iz), (i1 + 1, i2, iz), (i1, i2 + 1, iz),
+                 (i1 + 1, i2 + 1, iz)]
```

Comment:
For Python code, our [style guide](http://docs.openmc.org/en/latest/devguide/styleguide.html#python) says you need to follow [PEP8](https://www.python.org/dev/peps/pep-0008/). One rule there is that you should have a space around `=` in an assignment. It also says continuation lines should be lined up based on context above. I'd also change the variable name here to `indices`:
```Python
indices = [(i1, i2, iz), (i1 + 1, i2, iz), (i1, i2 + 1, iz),
           (i1 + 1, i2 + 1, iz)]
```